Src/Fare.Benchmarking/Benchmark.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using BenchmarkDotNet.Attributes;
+
+namespace Fare.Benchmarking
+{
+    public class Benchmark
+    {
+        private const string Pattern = ".";
```

Comment:
```suggestion
```

I often find the [rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) a reasonable metric when deciding to introduce a local variable. In this case we have only two usages, next to each other, so I think we may as well inline the regex.

Reply:
👍🏻 - no objections 😊